### ISSUE TITLE
random_string: ^2.0.1 package does not work 

### ISSUE THREAD

**Issue by SAMYAK99** [damondouglas/random_string.dart#10]
I have
-> added this package - random_string: ^2.0.1  in pubsec.yaml file 
-> pub.get
->import 'package:random_string/random_string.dart';  [this line gives error]

I have attached the images --- 
![randomString](https://user-images.githubusercontent.com/54956876/82482455-83134f80-9af4-11ea-96c6-2641ed85f8b1.JPG)

**Comment by jamesderlin** [comment]
It works for me.  Are you sure `pub get` succeeded?

**Comment by SAMYAK99** [comment]
yes !!!

**Comment by damondouglas** [comment]
@SAMYAK99 thank you for submitting this issue.  Just checking in to see if you are still experiencing the problem.

**Comment by SAMYAK99** [comment]
Yes the error still exists !!!

**Comment by damondouglas** [comment]
@SAMYAK99 I've tested on a both a Debian and MacOS machine from the command-line as well as using Android Studio and was able to install and use the library.  Both the sample command-line and the flutter app behaved as expected.  I noticed you are perhaps building a flutter app given at least one of the imports showing in your screenshot.  Are you able to get success using the command `flutter pub get` from your command-line?
